src/main.cpp

```diff
@@ -4364,18 +4370,19 @@ std::string GetWarnings(const std::string& strFor)
     // Misc warnings like out of disk space and clock is wrong
     if (strMiscWarning != "")
     {
-        strStatusBar = strGUI = strMiscWarning;
+        strStatusBar = strMiscWarning;
+        strGUI += strGUI.empty() ? "" : uiAlertSeperator + strMiscWarning;
```

Comment:
Hmm.. I don't see your solution with curly brackets, can you post an example? IMO this line is readable. If not, we could with to an if/else.